league_table/urls.py

```diff
@@ -0,0 +1,273 @@
+    #    name    = 'topic_list',
+    #),
+
+    #url(
+    #    regex   = '^topics/race-latino/?$',
+    #    view    = RedirectView.as_view(url=reverse_lazy('topic_detail', kwargs={'topic_slug': 'race-hispanic'})),
+    #    name    = 'topic_latino_redirect',
+    #),
+
+    #url(
+    #    regex   = '^topics/(?P<topic_slug>[-\w]+)/$',
+    #    view    = cache_page(STANDARD_CACHE_TIME)(TopicView.as_view()),
+    #    kwargs  = {},
+    #    name    = 'topic_detail',
+    #),
+
+    url(
+        regex   = '^examples/(?P<example_slug>[-\w]+)/$',
+        view    = cache_page(STANDARD_CACHE_TIME)(ExampleView.as_view()),
+        kwargs  = {},
+        name    = 'example_detail',
+    ),
+
+    #url(
+    #    regex   = '^glossary/$',
+    #    view    = cache_page(STANDARD_CACHE_TIME)(TemplateView.as_view(template_name="glossary.html")),
+    #    kwargs  = {},
+    #    name    = 'glossary',
+    #),
+
+    url(
+        regex   = '^locate/$',
+        view    = cache_page(STANDARD_CACHE_TIME)(LocateView.as_view(template_name="locate/locate.html")),
+        kwargs  = {},
+        name    = 'locate',
+    ),
+
+    url(
+        regex   = '^healthcheck$',
+        view    = HealthcheckView.as_view(),
+        kwargs  = {},
+        name    = 'healthcheck',
+    ),
+
+    url(
+        regex = '^robots.txt$',
+        view = lambda r: HttpResponse(
+            "User-agent: *\nDisallow: /api/\n%s: /" % ('Disallow' if getattr(settings, 'BLOCK_ROBOTS', False) else 'Allow') ,
+            content_type="text/plain"
+        )
+    ),
+
+    url(
+        regex   = '^place-search/json/$',
+        view    = PlaceSearchJson.as_view(),
+        kwargs  = {},
+        name    = 'place_search_json',
+    ),
+
+    # url for all schools page
+    url(
+        regex = '^schools/$',
+        view = schools,
+        kwargs = {},
+        name = 'schools',
+    ),
+
+    # url for specific school page
+    url(
+        regex = '^schools/(?P<code>[a-zA-Z0-9_-]+)/$',
+        view = specific_school,
+        kwargs = {},
+        name = 'specific_schools',
+    ),
+
+    # url for embedding league table
+    url(
+        regex = '^embed/(?P<geo_level>[a-zA-Z]+)/(?P<geo_code>[a-zA-Z0-9_-]+)$',
+        view = embed,
+        kwargs = {},
+        name = 'embed',
+    ),
+
+    # url for embedding the map
+    url(
+        regex   = '^embed/(?P<geography_id>\w+-\w+)(-(?P<slug>[\w-]+))?/$',
+        view    = cache_page(STANDARD_CACHE_TIME)(EmbedGeographyDetailView.as_view()),
+        kwargs  = {},
+        name    = 'embed_map',
+    ),
+
+    # LOCAL DEV VERSION OF API ##
+    # url(
+    #     regex   = '^geo-search/$',
+    #     view    = GeoSearch.as_view(),
+    #     kwargs  = {},
+    #     name    = 'geo_search',
+    # ),
+    #
+    # url(
+    #     regex   = '^elasticsearch/$',
+    #     view    = Elasticsearch.as_view(),
+    #     kwargs  = {},
+    #     name    = 'elasticsearch',
+    # ),
+    # END LOCAL DEV VERSION OF API ##
+)
```

Comment:
Is there a simpler way to add urls @phillipahereza ?

Reply:
@tebajanga why don't you import everything from `wazimap.urls.py` into this file and append you additional urls instead of duplicating everything `wazimap.urls.py` cc @DavidLemayian

Reply:
@philipahereza - Let me import from `wazimap.urls.py`